### PR TITLE
BB8-10943: Update the 2FA messaging generic for different scenarios

### PR DESCRIPTION
**Pull request by aagam-shah**
## Description

If a customer [enforced 2fa for all roles](https://docs.wpvip.com/security/two-factor-authentication/#h-enforce-two-factor-authentication-for-all-users), like subscriber, the 2FA admin notice banner we present to these lower cap users can be confusing/misleading.

This PR tries updating the message to keep it generic for different scenarios.

### CODE REVIEW

File: two-factor.php [hunk 1]
```diff
@@ -248,14 +248,14 @@ function wpcom_vip_two_factor_admin_notice() {
 			<div class="dashicons dashicons-warning" style="display:flex;float:left;margin-right:2rem;font-size:38px;align-items:center;margin-left:-20px;color:#ffb900;"></div>
 			<div>
 				<p style="font-weight:bold; font-size:16px;">
-					<a href="https://wpvip.com/documentation/vip-go/two-factor-authentication-on-vip-go/">Two Factor Authentication</a> is required to edit content on this site.
+					Access requires your account to have <a href="https://wpvip.com/documentation/vip-go/two-factor-authentication-on-vip-go/">Two-factor authentication</a> enabled.
 				</p>
 
-				<p>For the safety and security of this site, your account access has been downgraded. Please enable two-factor authentication to restore your access.</p>
+				<p>For the safety and security of this site, your account access has been downgraded. Please enable Two-factor authentication to restore your access.</p>
```
Comment (GaryJones) [inline two-factor.php]:
```suggestion
				<p>For the safety and security of this site, your account access has been downgraded. Please enable two-factor authentication to restore your access.</p>
```

File: two-factor.php [hunk 2]
```diff
@@ -248,14 +248,14 @@ function wpcom_vip_two_factor_admin_notice() {
 			<div class="dashicons dashicons-warning" style="display:flex;float:left;margin-right:2rem;font-size:38px;align-items:center;margin-left:-20px;color:#ffb900;"></div>
 			<div>
 				<p style="font-weight:bold; font-size:16px;">
-					<a href="https://wpvip.com/documentation/vip-go/two-factor-authentication-on-vip-go/">Two Factor Authentication</a> is required to edit content on this site.
+					Access requires your account to have <a href="https://wpvip.com/documentation/vip-go/two-factor-authentication-on-vip-go/">Two-factor authentication</a> enabled.
```
Comment (GaryJones) [inline two-factor.php]:
```suggestion
					Your account requires <a href="https://wpvip.com/documentation/vip-go/two-factor-authentication-on-vip-go/">two-factor authentication</a> to be enabled.
```

Comment (GaryJones) [reply two-factor.php]: While we know the plugin is called Two Factor Authentication, the customer doesn't need to treat it as a proper noun.

File: two-factor.php [hunk 3]
```diff
@@ -248,14 +248,14 @@ function wpcom_vip_two_factor_admin_notice() {
...
 				</p>
 
-				<p>For the safety and security of this site, your account access has been downgraded. Please enable two-factor authentication to restore your access.</p>
+				<p>For the safety and security of this site, your account access has been downgraded. Please enable Two-factor authentication to restore your access.</p>
 
 				<p>
 					<a href="<?php echo esc_url( admin_url( 'profile.php#two-factor-options' ) ); ?>" class="button button-primary">
-						Enable Two-factor Authentication
+						Enable Two-factor authentication
```
Comment (GaryJones) [inline two-factor.php]:
```suggestion
						Enable two-factor authentication
```

Comment (aagam-shah) [reply two-factor.php]: Since it's a button, I feel it's better to have T in uppercase

File: two-factor.php (hunk 2 above)
Comment (aagam-shah) [reply two-factor.php]: That's true, thanks for the suggestion

File: two-factor.php (hunk 3 above)
Comment (GaryJones) [reply two-factor.php]: In which case the A should be as well.

Comment (aagam-shah) [reply two-factor.php]: That's true. Updated in [`4e18d4b`](https://github.com/Automattic/vip-go-mu-plugins/pull/5133/commits/4e18d4bef0450b7f403ad8c8d28251d9f59793ad)

File: two-factor.php [hunk 4]
```diff
@@ -248,7 +248,7 @@ function wpcom_vip_two_factor_admin_notice() {
 			<div class="dashicons dashicons-warning" style="display:flex;float:left;margin-right:2rem;font-size:38px;align-items:center;margin-left:-20px;color:#ffb900;"></div>
 			<div>
 				<p style="font-weight:bold; font-size:16px;">
-					<a href="https://wpvip.com/documentation/vip-go/two-factor-authentication-on-vip-go/">Two Factor Authentication</a> is required to edit content on this site.
+					Your account requires <a href="https://wpvip.com/documentation/vip-go/two-factor-authentication-on-vip-go/">two-factor authentication</a> to be enabled.
```
Comment (rinatkhaziev) [inline two-factor.php]:
```suggestion
					Your account requires <a href="https://docs.wpvip.com/security/two-factor-authentication/">two-factor authentication</a> to be enabled.
```

**Review by rinatkhaziev — CHANGES_REQUESTED** [review]
@aagam-shah thanks! I noticed the link resulted in a redirect from the main site to docs site, so I copy-pasted the final link in the suggestion.

**Review by rinatkhaziev — APPROVED** [review]

**Pull request merged**